src/commands/ping.civet

```diff
@@ -0,0 +1,14 @@
+{ SlashCommandBuilder, ChatInputCommandInteraction, CacheType } from discord.js
+
+data := new SlashCommandBuilder()
+		.setName "ping"
+		.setDescription "Replies with Pong!"

```

Comment:
I don't see the purpose of this command... Is it to check whether the bot is running? It seems like `stats`/`versions` would satisfy that fine.